commit ae464a64a7f1f78ffcd720fe69798d44325e23d0 (refs/changes/60/760/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-20T17:01:09-07:00 (3 years ago)
    
    TOOLS-1584 imgapi client listImages() 'filters.channel' arg should win over 'this.channel' (bump ver and changelog)

diff --git a/CHANGES.md b/CHANGES.md
index f7292b0..216ec15 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -12,6 +12,11 @@
 
 ## not yet released
 
+## 10.0.2
+
+- TOOLS-1584 Ensure 'filters.channel' passed to IMGAPI.listImages wins over a
+  channel set on the IMGAPI instance.
+
 ## 10.0.1
 
 - IMGAPI-596 Move from restify-clients fork to restify-clients@1.4.0 with
diff --git a/package.json b/package.json
index d1df391..f93cf6d 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-clients",
   "description": "node.js client libraries for Triton core REST APIs.",
-  "version": "10.0.1",
+  "version": "10.0.2",
   "homepage": "http://www.joyent.com",
   "repository": {
     "type": "git",
